content/docs/user-guide/testing/best-practices/_index.md

```diff
@@ -89,14 +89,20 @@ Negative assertions are different from negative test cases. A negative test case
 
 It can be okay to include negative assertions when there is also at least one positive assertion, but they should not be relied on alone. Focus on identifying the expected output.
 
+### Disabling and skipping tests
+
+Ideally, all automated test failures prompt immediately fixing the broken feature (or updating the test when underlying expectations have changed). However, sometimes a tradeoff must be made to break a feature while enabling other functionality. In a case where functionality is known to be broken, automated tests can be temporarily "skipped" or disabled.
+
+Avoid immediately disabling tests when you see a failure. First consider what action the failure blocks, and why all contributors ignoring failure would be less risky. Whenever a test provided with O3DE gets disabled, please [cut an issue](https://github.com/o3de/o3de/issues/new/choose) to track fixing the feature and re-enabling the test.
+
 ## Practices to remember
 
 ### Insanity checks
 
-Ensure a new test is configured to run and report failures by temporarily editing the production code. Intentionally break the code in one way the test should detect. Then, run the test suite. If no failure occurs, investigate why!
+Ensure a new test is configured to run and report failures by temporarily editing the production code. Intentionally break the code in one way the test should detect, then run the test suite. If no failure occurs, investigate why!
```

Comment:
Is there a reason to prefer "a" to "one"? Should is appropriate, as it is not known whether the failure will be detected. Adding "that" and "and"

Reply:
It's hard to tell the difference, so one way we can look at this is by completely taking away "that the test can detect" and comparing the two versions: 

- "Intentionally break the code in one way, and then run the test suite"
- "Intentionally break the code in a way, and then run the test suite"

Now there's a greater contrast between the two sentences. The first version implies a singular or specific way to break the code. However, we don't mean to suggest that there's only one way to break the code, nor are we giving an example of one way to break the code. Looking at the second version, this doesn't make much sense - _in what way?_ This is incomplete because it needs a modifier, like "that the test can detect". So what "a" does is it adds generality. It suggests that there are can be one or multiple ways to do this -- whatever way is used, all that matters is that the broken code is one that the test can detect.